Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst

```diff
@@ -271,6 +271,8 @@ tt_content field.
 
 Another example shows the connection to a foreign table. This allows you to be more flexible with the possible values in the select box. The new field *tx_examples_main_category* is a connection to the TYPO3 system category table *sys_category*.
 
+.. code-block:: php
```

Comment:
You can replace this with a double colon (`::`) at the end of the previous paragraph since PHP code is already the default in this manual. (See other examples in this file)

Reply:
Thanks for the hint, updated the PR